tests/fsharpqa/Source/CodeGen/EmittedIL/SerializableAttribute/env.lst

```diff
@@ -1,5 +1,5 @@
-	SOURCE=ToplevelModule.fs    SCFLAGS="-a -g --out:desktop\\TopLevelModule.dll --test:EmitFeeFeeAs100001 --optimize-" COMPILE_ONLY=1 POSTCMD="..\\CompareIL.cmd ToplevelModule.dll NetFx20 desktop"		# ToplevelModule.fs - Desktop
```

Comment:
These tests were a bit messed up, just removing this `desktop` stuff as part of this PR